src/main/java/com/welovecoding/nbeditorconfig/processor/EditorConfigProcessor.java

```diff
@@ -43,7 +43,8 @@
   public EditorConfigProcessor() {
     // Project nbProject = OpenProjects.getDefault().getMainProject();
     Lookup lookup = Utilities.actionsGlobalContext();
-    Project project = lookup.lookup(Project.class);
+//    Project project = lookup.lookup(Project.class);
```

Comment:
Commented code can be removed

Reply:
I think there is more to that. If you look at the code, I initialize project to `null` then only test it to see if it's `null`. Quite useless. I cannot remember why I did this, it's a bit old now. Perhaps this is why I didn't create the PR. I'll have to complete that tonight.